### DISCUSSION TITLE
Playlist empty

### DISCUSSION THREAD

**Discussion by bnilsson11** [rern/rAudio#202] [Fixed Issues]
Hi,

Just installed rAudio for the first time, works fine, except for one thing:
The rightmost tab pane in the GUI of Library/Playback/Playlist is always empty, even though I am happily playing music.
I am playing both through UPnP rendering and via the normal rAudio GUI with USB DISK and NAS.
Previous RuneAudio versions showed the track list here, as I was adding tracks to play.
Have I missed something fundamental?

BN

**Comment by bnilsson11** [comment]
Web radios shows up, but not regular music tracks. Strange.

**Comment by rern** [comment]
Post result of this command:
`mpc playlist -f %album%^^%albumartist%^^%artist%^^%file%^^%time%^^%title%^^%track%`

**Comment by bnilsson11** [comment]
Today it suddenly worked, no intervention from me.
Case closed for the time being.
I will apply your instructions if the problem comes back.
Thanks.
